rudder-core/src/main/scala/com/normation/rudder/domain/appconfig/RudderWebProperty.scala

```diff
@@ -53,4 +57,29 @@ case class RudderWebProperty(
     name       : RudderWebPropertyName
   , value      : String
   , description: String
-)
\ No newline at end of file
+)
+
+
+/**
+ * A little domain language for feature switches
+ * (just enabled/disabled with the parsing)
+ */
+sealed trait FeatureSwitch { def name: String }
+object FeatureSwitch {
+
```

Comment:
The feature switch from the new search bar should used that one (it's in rudder core, the other is in web, we don't need two)